test/valid_schemas/draft2019-09/example.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
do we want to put this in a custom_tests directory or something to make this more obvious as to what it is?

Reply:
The `invalid_schemas` directory could go in there too I guess. That's a weird hold-over from the original maintainer.

Reply:
Are you thinking of putting this in `test/custom_tests/invalid_schemas`?